packages/fontaine/test/index.spec.ts

```
@@ -1,301 +1,81 @@
+            expect(typeof parse).toBe('function');
+            // Mocking a simple parse call check
+            const mockParseResult = { a: 'b' };
+            (parse as jest.Mock).mockReturnValue(mockParseResult);
+            
+            const result = parse('css string');
+            expect(parse).toHaveBeenCalled();
+            expect(result).toEqual(mockParseResult);
+        });
+
+        it('should expose the render function', () => {
+            expect(typeof render).toBe('function');
```

ESLint

[error] Extra semicolon.

(style/semi)